This patch disables tests that fail in github-ci, but pass on other linux systems.

--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -706,9 +706,7 @@
 	dbus/dynamic-method.test \
 	dbus/enum-string-marshalling.vala \
 	dbus/signals.test \
-	dbus/filedescriptor.test \
 	dbus/filedescriptor-async.test \
-	dbus/filedescriptor-errors.test \
 	dbus/dicts.test \
 	dbus/bug596862.vala \
 	dbus/bug602003.test \
